README.md

```diff
@@ -19,6 +19,8 @@ The repository builds to a static site deployed at [about.publiccode.net](https:
 
 Releasing and deploying [is done with a release branch](activities/documentation/merge-develop-into-master.md).
 
+If you want to make changes to the homepage on the about. website, you need to make changes in the file index.md . 
```

Comment:
You can use MarkDown for highlighting code, we tend to put filenames in back ticks as well.

Additionally, the Travis automated tester [has surfaced](https://travis-ci.com/github/publiccodenet/about/builds/214184654) that there is a surplus space at the end of the rule, so we can remove that.

```suggestion
If you want to make changes to the homepage on the about. website, you need to make changes in the file `index.md`.
```